src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
```

Comment:
According to the checklist, avoid using static methods in your solution. Instead of making 'getRandomBall()' a static method, you should create an instance of the Lottery class and use it to call the method, which you are already doing correctly here.